### ISSUE TITLE
Too much class

### ISSUE THREAD

**Issue by YifBookchainer** [KirbyZX/Level-0#6]
There are too many classes and functions in this program, as well as too may files, and overly spacey programming. It makes it hard to follow for people without pycharm, and is basically those who have pycharm opressing those who don't. Either way, I have some simple solutions: Lists, dictionaries and basic logic. I hope you understand and remove these issues as soon as possible.

Thanks,
A Concerned Citizen

**Comment by YifBookchainer** [comment]
Wow, I agree!

**Comment by kirbyzx** [comment]
> Class is gud.

- Lucas Kirby, 2k17
